### ISSUE TITLE
Do I have to reset my account even though I'm able to login into Keybase in my browser?

### ISSUE THREAD

**Issue by vwormer** [keybase/keybase-issues#3689]
Hello everyone
I recently deleted the Keybase app from my mobile device. This was the only device I used to use with Keybase. Now I'm not able to add any other device and it looks like Keybase is forcing me to reset my account. I have been using Keybase for years now and wonder if I really need to reset my full account just because I deleted the app on a particular device. When I log into Keybase in my browser, is there any way I can add a new device or reactivate the old one? 
Thanks a lot.

**Comment by heronhaye** [comment]
Yes, sorry. Because your private keys are only on your devices and unknown to us, you need to reset your account.

In the future, if you only have one electronic device, you should make a paper key as a backup in the Devices tab.
